packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -10,13 +10,10 @@
 
 %else
 %define repo_dir %{_sysconfdir}/yum.repos.d
-
-%if 0%{?rhel}
-%define repo_dist el%{rhel}
-%endif
+%define repo_dist %{dist}
```

Comment:
That's not the same tho.

```
[root@0f95f2b4985d /]# rpm -E '%dist'
.el7_9
[root@0f95f2b4985d /]# rpm -E '%rhel'
7
```

Reply:
https://github.com/theforeman/foreman-packaging/commit/f515215ed3d45bfd6d0d387bf0d10a08fa1084cf

Reply:
Looking at katello-repos it does look like for EL 8 & EL 9 it is correct, so it's just for EL 7 that it's "broken"?

Reply:
Today, yes.
Once RHEL8 becomes "LTS" I expect it to get there too.

Reply:
They really hate people who maintain repositories, don't they?